packages/create-nx-workspace/src/internal-utils/prompts.ts

```diff
@@ -80,6 +84,7 @@ export async function determineDefaultBase(
           message: `Main branch name`,
           initial: `main`,
           type: 'input',
+          skip: !parsedArgs.interactive || isCI(),
```

Comment:
Wouldn't it be easier to return without prompting if interactive is false?

Reply:
changed